lib/rubocop/cop/gemspec/required_ruby_version.rb

```diff
@@ -49,14 +57,15 @@ module Gemspec
       class RequiredRubyVersion < Base
         include RangeHelp
 
-        NOT_EQUAL_MSG = '`required_ruby_version` (%<required_ruby_version>s, ' \
-                        'declared in %<gemspec_filename>s) and `TargetRubyVersion` ' \
+        RESTRICT_ON_SEND = %i[required_ruby_version=].freeze
+        NOT_EQUAL_MSG = '`required_ruby_version` and `TargetRubyVersion` ' \
                         '(%<target_ruby_version>s, which may be specified in ' \
                         '.rubocop.yml) should be equal.'
         MISSING_MSG = '`required_ruby_version` should be specified.'
+        EXTRA_MSG = '`required_ruby_version` is set more than once.'
 
-        # @!method required_ruby_version(node)
-        def_node_search :required_ruby_version, <<~PATTERN
+        # @!method required_ruby_version?(node)
+        def_node_search :required_ruby_version?, <<~PATTERN
           (send _ :required_ruby_version= $_)
```

Comment:
This capture looks unnecessary.

```suggestion
          (send _ :required_ruby_version= _)
```

Reply:
Good call, it was left over from the previous version but I've removed it now 👍